docs/pages/admin-guides/deploy-a-cluster/separate-proxy-service-endpoints.mdx

```diff
@@ -0,0 +1,184 @@
+- A load balancer for traffic from Teleport Agents in private networks
+
+Each load balancer has a separate IP address. Both load balancers route traffic
+to the same pool of Teleport Proxy Service instances, such as the same AWS
+target group.
+
+Clients then connect to the Proxy Service load balancer at the appropriate
+address. Teleport Agents establish SSH reverse tunnels to the Proxy Service by
+dialing the internal load balancer, while external clients connect to the Proxy
+Service using the external load balancer. 
+
+When an end user connects to an infrastructure resource, e.g., using `tsh proxy
+db` or sending an HTTP request to a Teleport-protected application, they dial
+the external load balancer, which forwards the traffic to the Teleport Proxy
+Service. The Proxy Service then routes the traffic to the appropriate
+infrastructure resource by selecting a reverse tunnel.
+
+## Routing traffic to the appropriate load balancer
+
+Assuming you have deployed two load balancers, one for external Proxy Service
+clients and one for internal clients, there are three ways strategies for
+routing traffic to the appropriate load balancer from Teleport Agents and end
+users:
+
+- [Split DNS](#split-dns)
+- [Multiple public addresses](#multiple-public-addresses)
+- [Tunnel public address](#tunnel-public-address)
+
+### Split DNS
+
+You can set up your infrastructure to use split DNS so that a single public
+address for your Teleport cluster resolves to the external endpoint for end
+users and the internal endpoint for infrastructure resources in your internal
+network.
+
+1. Determine the public address to use for the Teleport Proxy Service, e.g.,
+   `proxy.example.com`.
+
+1. Configure your DNS infrastructure to return the IP address of the internal
+   load balancer for your internal network and the external load balancer for
+   the public internet.
+
+   For example, in Amazon Route 53, you can create public and private hosted
+   zones for the same domain and add a separate record to each hosted zone ([AWS
+   documentation](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zone-private-considerations.html#hosted-zone-private-considerations-split-view-dns)).
+
+1. Edit your Proxy Service configuration to include a single value for its
+   public address:
+
+   ```yaml
+   proxy_service:
+     enabled: true
+     public_addr: proxy.example.com:3080
+   ```
+
+### Multiple public addresses
+
+You can configure the Teleport Proxy Service to advertise multiple addresses on
+its SSH host certificate. This enables you to instruct internal clients (e.g.,
+Teleport Agents) to connect to the Proxy Service on the internal address while
+external clients connect on the external address. 
+
+1. Configure the Teleport Proxy Service to use multiple public address values:
+
+   ```yaml
+   proxy_service:
+     enabled: true
+     public_addr: 
+       - external_proxy.example.com:3080
+       - internal_proxy.example.com:3024
```

Comment:
The tunnel endpoint generally still uses port :3080 (or more commonly 443) because in this example multiplexing is still enabled and there is no separate tunnel listener.